cache

```diff
@@ -828,6 +828,7 @@ cache::check_lftp() {
     return 0;
   else
     cache::err "The 'lftp' executable is missing or not in the \$PATH"
+    exit 1
```

Comment:
I suggest implementing a `cache::fatal "The 'lftp' executable is missing or not in the \$PATH"` method.

Easier to read and maintain.